config/settings.py

```diff
@@ -28,51 +28,58 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = env("DEBUG", default=False)
 
-ALLOWED_HOSTS = env.list("ALLOWED_HOSTS", default=[])
+ALLOWED_HOSTS = env("ALLOWED_HOSTS").split(",")
 
 
 # Application definition
 
 INSTALLED_APPS = [
-    'django.contrib.admin',
-    'django.contrib.auth',
-    'django.contrib.contenttypes',
-    'django.contrib.sessions',
-    'django.contrib.messages',
-    'django.contrib.staticfiles',
-
-    'kanban',
+    "user_account",
```

Comment:
next time, if you run a formatter on a whole code base, you ideally want to make a separate PR for that, because it makes it really hard to discern what was actually changed or not

Reply:
I know, I didn't mean that, it was an accident, sorry. :(